pkg/systemd/service-details.jsx

```diff
@@ -502,7 +502,7 @@ export class ServiceDetails extends React.Component {
                                     <React.Fragment key={rel.Name.split().join("")}>
                                         <label className="control-label closer-lines" htmlFor={rel.Name}>{rel.Name}</label>
                                         <ul id={rel.Name.split(" ").join("")} className="comma-list closer-lines">
-                                            {rel.Units.map(unit => <li key={unit}><a href={"#/" + unit} className={this.props.isValid(unit) ? "" : "disabled"}>{unit}</a></li>)}
+                                            {rel.Units.map(unit => <li key={unit}><a href={"#/" + unit} disabled={!this.props.isValid(unit)}>{unit}</a></li>)}
```

Comment:
hmm, this does not seem to actually disable it. https://css-tricks.com/how-to-disable-links/

Reply:
Nice to know, and oops, I missed the <a> thingy there.

Reply:
The point here is close to the li argument, it definitely does not respect disabled attribute, but even if you remove the href prop, you'd get the link treatment.